extensions/amp-viqeo-player/0.1/test/validator-amp-viqeo-player.html

```diff
@@ -0,0 +1,36 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Please follow https://github.com/ampproject/amphtml/blob/master/examples/amp-video.amp.html so we can easily test actions and autoplay behaviour.

Reply:
I've made the similar test page